app.py

```diff
@@ -261,7 +262,10 @@ def autoscale(self):
                 '<meta property="og:image:type" content="image/png" />',
                 '<meta property="og:image:height" content="1114" />'
                 '<meta property="og:image:width" content="1112" />',
+                *(trackers if os.getenv("ENABLE_TRACKERS") else []),
```

Comment:
can we import `ENABLE_TRACKERS` from [CONST.py](https://github.com/Lightning-AI/stable-diffusion-deploy/blob/main/muse/CONST.py#L4) please?